model/PasswordResetResponse.ts

```diff
@@ -0,0 +1,4 @@
+export interface PasswordResetResponse {
+	status: string
+	url: string
+}
```

Comment:
if it has a failed status it could have a field called issues, in mpay looks like this:
public abstract class MPayProcessResponse {

    @Schema(description = "If status is not SUCCESS, contains the errors")
    private List<@NotNull Issue> issues;

    @Schema(description = "Status of this response")
    private Status status;



    /**
     * The process status can be either SUCCESS or FAILURE.
     * <p>
     */
    public enum Status {
        SUCCESS, FAILURE
    }

    /**
     * Enum for severity level of a noted issue.
     * <p>
     */
    public enum Severity {
        @Deprecated
        INFO,
        @Deprecated
        WARNING,
        ERROR,
        FATAL
    }

    /**
     * An issue note from executing a process.
     * <p>
     */
    @Builder
    @Getter
    @Setter
    @Valid
    @ToString
    @EqualsAndHashCode
    @NoArgsConstructor
    @AllArgsConstructor
    public static class Issue {

        @Schema(description = "Severity of the issue. Always fatal or error.", deprecated = true)
        @NotNull
        private Severity severity;

        @Schema(description = "Error message")
        @NotBlank
        private String message;

    }